### DISCUSSION TITLE
Is there a way for the "wheelup" and "wheeldown" to be faster?

### DISCUSSION THREAD

**Discussion by ghost** [kriomant/ch57x-keyboard-tool#104] [Q&A]
I'm trying to use one of the Macropad keyboards to help with scrolling. The "wheelup" and "wheeldown" are slow. I tried "pageup" and "pagedown", but that's too fast. Is there a place where I can edit the code to make my own macro of sorts?

**Comment by kriomant** [comment]
Please check whether it is possible using official software.
